cmd/mps-control-daemon/mount/mount-shm.go

```diff
@@ -17,26 +17,46 @@
 package mount
 
 import (
+	"bufio"
 	"fmt"
 	"os"
 	"os/exec"
+	"strconv"
+	"strings"
 
 	"github.com/urfave/cli/v2"
+	"k8s.io/klog/v2"
 	"k8s.io/mount-utils"
 )
 
+type options struct {
+	shmSize string
+}
+
 // NewCommand constructs a mount command.
 func NewCommand() *cli.Command {
-	// Create the 'generate-cdi' command
-	return &cli.Command{
-		Name:   "mount-shm",
-		Usage:  "Set up the /dev/shm mount required by the MPS daemon",
-		Action: mountShm,
+	opts := options{}
+
+	c := cli.Command{
+		Name:  "mount-shm",
+		Usage: "Set up the /dev/shm mount required by the MPS daemon",
+		Action: func(ctx *cli.Context) error {
+			return mountShm(ctx, &opts)
+		},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "shm-size",
+				Destination: &opts.shmSize,
+				Hidden:      true,
```

Comment:
What does `hidden` do?

Reply:
It doesn't show it in the help text. I can remove this if required but I was still half-way between updating the default and allowing a user to configure this as per #699

Reply:
I have removed this option since it's not used.